### ISSUE TITLE
Calling semantic-release do nothing

### ISSUE THREAD

**Issue by ghaiklor** [semantic-release/cli#75]
Calling `semantic-release setup` do nothing and just shows help.
Version 1.4.0

```shell
~/L/P/generator-sails-rest-api   dev   semantic-release-cli setup

semantic-release-cli

Usage:
  semantic-release-cli setup [--tag=<String>]

Options:
  -h --help            Show this screen.
  -v --version         Show version.
  --[no-]keychain      Use keychain to get passwords [default: true].
  --ask-for-passwords  Ask for the passwords even if passwords are stored [default: false].
  --tag=<String>       npm tag to install [default: 'latest'].
  --gh-token=<String>  Github auth token
  --npm-token=<String> NPM auth token

Aliases:
  init                 setup
 ~/L/P/generator-sails-rest-api   dev  
```

**Comment by lewiscowper** [comment]
This looks like I broke it here: 854d716, but I can't see why, and nobody caught anything wrong when we saw it.

Installing the previous release with `npm install -g semantic-release-cli@1.3.3` does not have this regression.

I guess either reverting the aforementioned commit, or finding a fix within it is a good idea.

There are only two lines changed that would affect logic, the most obvious one that the bug could be in is: https://github.com/semantic-release/cli/blob/master/src/index.js#L52

Hope that vague debug helps, the quickest workaround would be installing `@1.3.3`, but we hope to figure it out and fix this asap!
